### ISSUE TITLE
Progressbar max_value parameter issue

### ISSUE THREAD

**Issue by slawchune** [OceanParcels/parcels#443]
Hello Eric, 

I have experienced an issue with progressbar, maybe due to my python environment (python 3.5, but progressbar 2.3 py27_0 ). 

It appears that the first parameter  'max_value' leads to an error and that "maxval" should be used instead. I have made the corrections for "particleset.py' : 

pbar = progressbar.ProgressBar(maxval=abs(endtime - _starttime)).start()

Could i have some feedback about this please ? 

Thank a lot,
Stéphane

**Comment by delandmeterp** [comment]
Hi Stéphane,

You are not the first one to experience this bug. I have progressbar 3.37.1 and everything goes fine for me. 
`maxval` is deprecated and `max_value` is used in newer versions of progressbar...

@erikvansebille, we should update the Parcels requirement list such that a minimum version of progressbar is required.

**Comment by slawchune** [comment]
thanks a lot for your answer Phillipe !

**Comment by erikvansebille** [comment]
Reopening because this will need to be fixed indeed. 

We either require a minimum version number as you suggested @delandmeterp , or we could do something like
```
try:
    pbar = progressbar.ProgressBar(max_val=abs(endtime - _starttime)).start()
except:
    pbar = progressbar.ProgressBar(maxval=abs(endtime - _starttime)).start()
```

**Comment by delandmeterp** [comment]
Yes also. 
Anyway `maxval` still works, but we shouldn't use it because at some points it might not work anymore.

```
~ $ python
Python 2.7.15 |Anaconda, Inc.| (default, May  1 2018, 18:37:05) 
[GCC 4.2.1 Compatible Clang 4.0.1 (tags/RELEASE_401/final)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import progressbar
>>> progressbar.ProgressBar(maxval=3)
<progressbar.bar.ProgressBar object at 0x10a3642d0>
>>> 
100% (3 of 3) |###################################################################################| Elapsed Time: 0:00:00 ETA:  00:00:00
~ $ python
Python 2.7.15 |Anaconda, Inc.| (default, May  1 2018, 18:37:05) 
[GCC 4.2.1 Compatible Clang 4.0.1 (tags/RELEASE_401/final)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import progressbar, warnings
>>> warnings.filterwarnings("default", category=DeprecationWarning)
>>> progressbar.ProgressBar(maxval=3)
/Users/delandmeter/anaconda2/envs/py2_parcels/lib/python2.7/site-packages/progressbar/bar.py:240: DeprecationWarning: The usage of `maxval` is deprecated, please use `max_value` instead
  '`max_value` instead', DeprecationWarning)
<progressbar.bar.ProgressBar object at 0x10cb6a350>
>>> 
100% (3 of 3) |###################################################################################| Elapsed Time: 0:00:00 ETA:  00:00:00
~ $ 
```

**Comment by erikvansebille** [comment]
Fixed in #444
